pt/development/dispatch-filters.rst

```diff
@@ -1,12 +1,199 @@
+  como CSS, JavaScript ou arquivo de imagem armazenado na pasta raiz da web de um plug-in ou na pasta 
+  correspondente a um Tema. Ele servirá o arquivo de acordo, se encontrado, interrompendo o restante do 
+  ciclo de despacho::
+
+        // Use as opções para definir o cacheTime para seus arquivos estáticos
+        // Se não estiver definido, o padrão será +1 hora
+        DispatcherFactory::add('Asset', ['cacheTime' => '+24 horas']);
+
+* ``RoutingFilter`` aplica regras de roteamento de aplicativos a URL da solicitação.
+   Preenche ``$request->getParam()`` com os resultados do roteamento.
+* ``ControllerFactory`` usa ``$request->getParam()`` para localizar o controlador que
+   tratará da solicitação atual.
+* ``LocaleSelector`` permite a troca automática de idioma a partir do ``Accept-Language``
+   cabeçalho enviado pelo navegador.
+
+Usando Filtros
+==============
+
+Os filtros geralmente são ativados no arquivo **bootstrap.php** do seu aplicativo, mas você 
+pode carregá-los a qualquer momento antes do envio da solicitação. A adição e remoção de filtros 
+é feita através de :php:class:`Cake\\Routing\\DispatcherFactory`. Por padrão, o modelo de 
+aplicativo CakePHP vem com algumas classes de filtro já ativadas para todas as solicitações; 
+vamos dar uma olhada em como eles são adicionados::
+
+    DispatcherFactory::add('Routing');
+    DispatcherFactory::add('ControllerFactory');
+
+    // A sintaxe do plug-in também é possível
+    DispatcherFactory::add('PluginName.DispatcherName');
+
+    // Use as opções para definir a prioridade
+    DispatcherFactory::add('Asset', ['priority' => 1]);
+
+Filtros de expedidor com ``priority`` (prioridade) mais alta (números mais baixos) - serão executados primeiro. 
+O padrão de prioridade é ``10``.
+
+Embora o uso do nome da string seja conveniente, você também pode passar instâncias para ``add()``::
+
+    use Cake\Routing\Filter\RoutingFilter;
+
+    DispatcherFactory::add(new RoutingFilter());
+
+
+Configurando a Ordem dos Filtros
+--------------------------------
+
+Ao adicionar filtros, você pode controlar a ordem em que eles são chamados usando as 
+prioridades do manipulador de eventos. Embora os filtros possam definir uma prioridade 
+padrão usando a propriedade ``$_priority``, você pode definir uma prioridade específica ao 
+anexar o filtro::
+
+    DispatcherFactory::add('Asset', ['priority' => 1]);
+    DispatcherFactory::add(new AssetFilter(['priority' => 1]));
+
+Quanto maior a prioridade, mais tarde esse filtro será chamado.
+
+Aplicação condicional de filtros
+--------------------------------
+
+Se você não deseja executar um filtro em todas as solicitações, poderá usar condições 
+para aplicá-lo apenas algumas vezes. Você pode aplicar condições usando as opções ``for`` 
+e ``when``. A opção ``for`` permite que você combine com substrings de URL, enquanto a 
+opção ``when`` permite executar uma chamada:
```

Comment:
Missing a second : at the end of this line.

Reply:
Done